.travis.yml

```diff
@@ -49,6 +54,18 @@ jobs:
       script: ./gradlew publish
       if: branch = master AND type != pull_request
 
+    - stage: publish
```

Comment:
maybe call the stage publish_docker since there will come more publishing stages